src/docs/app-monitoring/sysdig-monitoring-guideline-for-shared-services-and-apps.md

```diff
@@ -0,0 +1,347 @@
+
+Key features of Uptime.com include:
+
+- **Availability Monitoring**: Checks if your site is accessible and notifies you immediately if it detects any downtime.
+- **Performance Monitoring**: Measures site speed, helping identify bottlenecks that could impact user experience.
+- **Domain Health Check**: Monitors SSL certificates, domain expirations, and more to ensure the health and security of your domain.
+- **Transactional Tests**: Simulates user paths and interactions on your site, ensuring critical processes like logins or shopping cart checkouts work flawlessly.
+
+By integrating tools like Uptime.com into the SRE toolkit, teams can proactively address issues, ensuring optimal user experience and meeting established Service Level Objectives (SLOs).
+
+We can set up a transactional test for the registry. This test continually loads the dashboard page to ensure all components are successfully loaded. Uptime.com provides insights into the uptime and downtime of your application and can display the percentage of uptime over a specified period. Additionally, it can send out notifications, enabling us to respond promptly.
+
+
+## Runbook
+To achieve 99.5% uptime, we have a daily allowance of just 7 minutes and 12 seconds for downtime. This is where RunBooks prove invaluable. In SRE, the objective is to automate as many processes as feasible. In the realm of cloud operations, Runbooks consist of a series of steps carried out by SREs to accomplish specific tasks. These tasks can encompass incident responses, cost management, addressing performance challenges, and more.
+
+### Runbook for Incident Response
+
+One of the primary duties of SREs is to respond to incidents. What actions might an SRE carry out during an outage? Here's a typical sequence:
+
+1. Trigger
+2. Troubleshooting
+3. Root cause analysis
+4. Fix
+
+Runbooks can include embedded logic, such as if-else statements and loops, as well as other functionalities beyond being a mere list of actions. For example, they can incorporate features like waiting for a resource. Essentially, a Runbook offers instructions on the steps to be taken or the processes for an automated system to follow when an SLO is breached. This ensures that we adhere to our SLA and avoid violations.
+
+For a simple example: 
+
+- When there are a few requests to the database that fail or timeout:
+  - Execute a command in our backup-container to save the current data in our database.
+  - Allocate more resources to the primary service.
+
+- When the dashboard load time nears our SLO:
+  - Scale up and terminate the old pod.
+  - Notify the development team to assess the network status.
+
+For more insights on automation runbooks, refer to this [source in Xenonstack](https://www.xenonstack.com/insights/automation-runbook-for-sre). We are also introducing [runwhen](https://www.runwhen.com/) on our platform to aid in this automation process.
```

Comment:
we don't use runwhen ourselves, but you could say here's a cool tool for that ;)